azext_iot/central/_help.py

```diff
@@ -319,6 +320,92 @@ def _load_central_api_token_help():
     """
 
 
+def _load_central_cde_help():
+    helps[
+        "iot central app cde"
+    ] = """
+        type: group
+        short-summary: Create and Manage Continuous Data Export (CDE) for an application.
+        long-summary: Continuous Data Export enables exporting IoT Central measurements, devices, and device templates data to storage account.
+    """
+
+    helps[
+        "iot central app cde create"
+    ] = """
+        type: command
+        short-summary: Create a new continuous data export
+        examples:
+        - name: Add new continuous data export
+          text: >
+            az iot central app cde create
+            --app-id {appid}
+            --display-name {displayName}
+            --sources "devices, deviceTemplates, telemetry"
+            --enable True,
+            --ep-type {epType}
+            --ep-conn {epConnStr}
+            --entity-name {entityName}
+            --export-id {exportId}
+    """
+
+    helps[
+        "iot central app cde show"
+    ] = """
+    type: command
+    short-summary: Get continuous data export by ID
+    examples:
+      - name: Get continuous data export information
+        text: >
+          az iot central app cde show
+          --app-id {appid}
+          --export-id {exportId}
+    """
+
+    helps[
+        "iot central app cde delete"
+    ] = """
+    type: command
+    short-summary: Delete a continuous data export from the application
+    examples:
+      - name: Delete a continuous data export
+        text: >
+          az iot central app cde delete
+          --app-id {appid}
+          --export-id {exportId}
+    """
+
+    helps[
+        "iot central app cde list"
+    ] = """
+    type: command
+    short-summary: Get the list of continuous data exports in an application.
+    examples:
+      - name: List of continuous data exports
+        text: >
+          az iot central app api-token list
```

Comment:
This example entry is invalid for this command (the other thing the command table linter raised).